### PR TITLE
docs: add 4.0 docs generation

### PR DESCRIPTION
**Pull request by nbbeeken**
docs

In addition to adding the 4.0 docs scripts I updated the front page a bit. The reference link for 4.0 goes to the docs team page.

### CODE REVIEW

**Review by durran — APPROVED** [review]
LGTM

File: site/layouts/partials/features.html [hunk 1]
```diff
@@ -2,15 +2,24 @@ <h2>Features</h2>
...
-  <dt>Core driver</dt>
-  <dd>The MongoDB Node.js core driver is the new underpinning for the
-	  driver and is meant for framework developers that do not need the
-	  helpers and abstractions available in the full driver. This driver
-	  is not meant for end users, as it offers none of the conveniences
-	  of the higher level APIs</dd>.
+  <dd>
+	Whether you are new to MongoDB or you just want to brush up on the core concepts, you can check out the Fundamentals section which features material on using the Node.js driver for the following:
+	<ul>
+		<li><a href="/drivers/node/fundamentals/connection">Connection Guide</a>: connect to a MongoDB
```
Comment (emadum) [inline site/layouts/partials/features.html]: Don't these need to point to off-domain to `https://docs.mongodb.com`?

**Review by emadum — COMMENTED** [review]
Just a question about the links, they seem broken [here](https://mongodb.github.io/node-mongodb-native/).

File: site/layouts/partials/features.html (hunk 1 above)
Comment (nbbeeken) [reply site/layouts/partials/features.html]: Ah good catch, did some reformatting and lost the full links, fixed!

**Pull request merged**